packages/common/src/errors.ts

```diff
@@ -32,4 +32,5 @@ export const enum RuntimeErrorCode {
   UNEXPECTED_DEV_MODE_CHECK_IN_PROD_MODE = 2958,
   INVALID_LOADER_ARGUMENTS = 2959,
   OVERSIZED_IMAGE = 2960,
+  OVERLAPPING_BREAKPOINTS = 2961
```

Comment:
We can probably add a comma at the end here as well (similar to last error code on the current list):
```suggestion
  OVERLAPPING_BREAKPOINTS = 2961,
```

Reply:
Removed this error case entirely since there's now only one breakpoint array.